### PR TITLE
DS-336 Add layer 3 overlay max width design token

### PR DESCRIPTION
**Pull request by marciopmoreira6**

### CODE REVIEW

File: design-tokens/tokens/layer3/base.json [hunk 1]
```diff
@@ -1 +1,14 @@
-{}
\ No newline at end of file
+{
+  "echoes": {
+    "sizes": {
+      "overlays": {
+        "maxWidth": {
+          "default": {
+            "value": "{echoes.dimension.space.800}",
```
Comment (gregaubert) [inline design-tokens/tokens/layer3/base.json:7]: @marciopmoreira6 it should reference `size.5000` right?
I'll fix it with a commit, I need to fix the build for layer 3 anyway.

File: design-tokens/build.js [hunk 2]
```diff
@@ -36,11 +36,14 @@ const licenseHeader = fs.readFileSync(`config/license/LICENSE-HEADER.txt`, 'utf-
 const designTokenGroups = JSON.parse(
   fs.readFileSync(`${DESIGN_TOKENS_PATH}/$themes.json`, 'utf-8'),
 );
+const baseDesignTokenGroup = designTokenGroups.filter(
+  ({ group, name }) => group === 'Sonar' && name === 'base',
+)[0];
```
Comment (jeremy-davis-sonarsource) [inline design-tokens/build.js:41]: NP:
```
const baseDesignTokenGroup = designTokenGroups.find(
  ({ group, name }) => group === 'Sonar' && name === 'base',
);
```

And maybe we should exit if we don't find it?

**Review by jeremy-davis-sonarsource — CHANGES_REQUESTED** [review]

File: design-tokens/tokens/layer3/base.json (hunk 1 above)
Comment (marciopmoreira6) [reply design-tokens/tokens/layer3/base.json]: thanks

File: design-tokens/build.js (hunk 2 above)
Comment (gregaubert) [reply design-tokens/build.js:41]: Good catch, I'll update it.

> And maybe we should exit if we don't find it?

I don't think it's worth going through the hassle of handling these cases, it shouldn't happen, if it does, it's not gonna generate the expected files and/or break and that's fine.

**Review by jeremy-davis-sonarsource — APPROVED** [review]

**Pull request merged**